docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -1,33 +1,32 @@
 # Zowe Server Install Wizard Overview
 
-The Zowe Server Install Wizard is an installation wizard for Zowe server-side components. You can install the wizard on Microsoft Windows, Mac OS, and Linux system. It is an alternative to the existing Zowe installation process and improves the user experience.
+The Zowe Server Install Wizard is an installation wizard for Zowe server-side components. You can install the wizard on Microsoft Windows, Mac OS, and Linux systems. Performing installation via the installation wizard streamlines the installation process and is an alternative to performing manual Zowe server-side component installation.
```

Comment:
"You can install the wizard..." maybe we can say "You can run the wizard..."? 
"Install" sounds like extra steps to do